### ISSUE TITLE
multiple errors on install of ipfs-npm: will not install through desktop client, cannot direct error report, alternative doesn't work either?!!?

### ISSUE THREAD

**Issue by Hyperpolymath** [ipfs-shipyard/ipfs-desktop#1688]
Please describe what you were doing when this error happened.

Tryng to install the ipfs-npm module via the IPFS desktop client, and then tried to do the report (BUT this then says the link is too long, and cannot be shortened by bit.ly or other link shortener)...

The tried to follow advice to install direct using the npm install -g approach, and that fails too.

**Specifications**

- **OS**: win32
- **IPFS Desktop Version**: 0.13.0
- **Electron Version**: 9.3.2
- **Chrome Version**: 83.0.4103.122

**Error**

```
TypeError: shell.openItem is not a function
    at recoverableErrorDialog (C:\Users\jimme\AppData\Local\Programs\IPFS Desktop\resources\app.asar\src\dialogs\errors.js:77:11)
    at Object.install (C:\Users\jimme\AppData\Local\Programs\IPFS Desktop\resources\app.asar\src\npm-on-ipfs\package.js:38:5)
    at processTicksAndRejections (internal/process/task_queues.js:94:5)
    at async IpcMainImpl.<anonymous> (C:\Users\jimme\AppData\Local\Programs\IPFS Desktop\resources\app.asar\src\utils\create-toggler.js:10:9)
```

**Comment by jessicaschilling** [comment]
Hi @Hyperpolymath - this is a known issue and is being actively worked in https://github.com/ipfs-shipyard/ipfs-desktop/pull/1687.

**Comment by lidel** [comment]
@Hyperpolymath 
We fixed log preview and error reporting in #1687 which will ship with v0.13.1 ships next week.
This should make it easier to see what went wrong with npm-on-ipfs.

If the problem with ipfs-npm experiment remains, see if your error is the same as in https://github.com/ipfs-shipyard/ipfs-desktop/issues/1504 before filling a new issue.
